api/db-cdr/build.gradle

```diff
@@ -15,14 +15,17 @@ def liquibase_password = System.getenv("LIQUIBASE_DB_PASSWORD") ?: "lb-notasecre
 
 liquibase {
     activities {
-        main {
-            changeLogFile "changelog/db.changelog-master.xml"
+        schema {
+            changeLogFile "changelog-schema/db.changelog-master.xml"
             url "jdbc:mysql://${db_host}:${db_port}/cdr"
             username "liquibase"
             password "${liquibase_password}"
         }
-        local {
-            changeLogFile "changelog-local/db.changelog-master.xml"
+        data {
+            if (project.hasProperty('contexts')) {
```

Comment:
When is this not the case?

Reply:
I agree, but the liquibase gradle plugin doesn't recognize contexts without this conditional.